Our-Project.ipynb

```diff
@@ -7,6 +7,111 @@
     "# This is the start!!!"
```

Comment:
<p>Shall we use the <code>final_genres</code>  variable from the first cell block instead of hardcoding the <code>top_genres</code>  array?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CS3244-ML-AY2021S1-0037/CS3244-ML/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>sure but that would be additional processing time</p>

Reply:
okay, i think it would be fine since its only run once?

Reply:
lets ignore this for now -- if there arent any other stuff pending, for this PR,  you can merge this in!